### PR TITLE
Abstract for Gamma 2016 poster contribution

### PR DESCRIPTION
**Pull request by cdeil**
I would like to make a poster for the Gamma 2016 conference to advertise the open-gamma-ray-astro effort to the gamma-ray community.

The conference is July 11-15, 2016 in Heidelberg.
Abstract submission deadline is May 22.
https://www.mpi-hd.mpg.de/hd2016/

cc @kialio @woodmd @cboisson @joleroi @jknodlseder @TarekHC

This is a pull request with the title and abstract. Please review and comment now.

I'll make a first proposal for the poster later and we can then use that as the starting point for discussions on content. Also we can decide whether to write a proceeding on this or not later.

For co-authors, my suggestion would be:
1. Put everyone that has made a contribution to the open spec or was present at the IACT DL3 meeting in Meudon and give them the option to opt-out.
2. Invite everyone on the [open-gamma-ray-astro](https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro) to co-author, i.e. make it opt-in for them (75 people).

### CODE REVIEW

File: abstract.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Open data formats and software for gamma-ray astronomy
+======================================================
+
+Authors: tbd
+
+In gamma-ray astronomy, a lot of proprietary data formats and software exist
+that were developed for one specific mission or experiment. Especially in
```
Comment (TarekHC) [inline abstract.txt]: In gamma-ray astronomy, a variety of data formats and proprietary software exist,
 developed for one specific mission or experiment.

File: abstract.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+ground-based gamma-ray astronomy, imaging atmospheric Cherenkov telescope (IACT)
+data and software until now was private to the collaborations operating the
```
Comment (TarekHC) [inline abstract.txt]: Maybe?  -> data and software has been private to the collaborations operating the telescopes

File: abstract.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+telescopes. There is a general movement in science towards open data and
+software and specifically the next big IACT, the Cherenkov telescope array (CTA)
```
Comment (TarekHC) [inline abstract.txt]: Cherenkov Telescope Array (CTA)

File: abstract.txt [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+will be operated as an open observatory.
+
+We have created a Github organisation at https://github.com/open-gamma-ray-astro
+where we are developing data format specifications. A public mailing list was
+set up at https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro and a
+first face-to-face meeting on IACT high-level data model and formats took place
+in April 2016 in Meudon. We hope that this open multi-mission effort will help
+accelerate the development of open data formats and open-source software for
+gamma-ray astronomy, leading to synergies in the development of analysis codes
+and eventually better science results (reproducible, multi-mission). This poster
```
Comment (TarekHC) [inline abstract.txt]: better scientific results

File: abstract.txt [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+will summarise what we have done so far and hopefully will solicit comments or
```
Comment (TarekHC) [inline abstract.txt]: I would remove the "hopefully" (we will solicit for comments for sure!)  :)

**Comment by TarekHC** [comment]
Hi Christoph, 

Thank you for taking the initiative and the nice abstract.

I have some minor modifications. I added them as notes into the code (not sure if this is the best approach...).

File: abstract.txt (hunk 1 above)
Comment (cdeil) [inline abstract.txt]: done

File: abstract.txt (hunk 2 above)
Comment (cdeil) [inline abstract.txt]: done

File: abstract.txt (hunk 3 above)
Comment (cdeil) [inline abstract.txt]: done

File: abstract.txt (hunk 4 above)
Comment (cdeil) [inline abstract.txt]: done

File: abstract.txt (hunk 5 above)
Comment (cdeil) [inline abstract.txt]: I changed from "... hopefullly will ..." to "... has the goal to ...":

> This poster will summarise what we have done so far and has the goal to solicit comments and
> future contributions from the gamma-ray astronomy community.

(I don't want to be presumptuous and claim "... will solicit comments ...")

**Comment by cdeil** [comment]
> I have some minor modifications.

@TarekHC - Thanks for the feedback. I've applied your comments in 12f3780.

> I added them as notes into the code (not sure if this is the best approach...).

Inline comments are OK.
Note that now that the lines on which you commented have changed, the Github web interface will hide them by default, but one can still click on it as see them.
This is the compromise Github chose between keeping the record of inline comments, but keeping the overall pull request sane.

(The alternative to inline comments would be just one normal comment listing suggestions, like you'd do e.g. via email, which is fine as well.)

**Comment by cboisson** [comment]
To Christoph: Thanks for the abstract  (with Tarek suggestions) advertising the "opening" of the DL3specifications. 

Your author list scheme is fine and may be the abstract could be send asap to the long list for opt-in as the dead line for abstract submission is May 22nd if I remember well.

**Comment by cdeil** [comment]
@TarekHC  and @cboisson - Thanks for the feedback!

I did put the author list as suggested in my initial description.
Merging this now.

I'll circulate this on https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro now.

**Pull request merged**